js/admin.js

```diff
@@ -159,4 +159,59 @@
         : checkConnection.css({ color: 'red' }).text('Connection failed');
```

Comment:
Two more errors when checking the Coinsnap connection:

<img width="1076" alt="image" src="https://github.com/user-attachments/assets/14ef65ae-72e8-46c4-82f7-2867b51471a7" />

Reply:
Pobably the check should be disabled if the fields are not populated:

<img width="753" alt="image" src="https://github.com/user-attachments/assets/2480d138-6448-462e-b6cc-102b33f16b65" />